client/packages/requisitions/src/ResponseRequisition/api/hooks/index.ts

```diff
@@ -23,6 +23,7 @@ export const useResponse = {
     api: Utils.useResponseApi,
     createOutbound: Utils.useCreateOutboundFromResponse,
     isDisabled: Utils.useIsResponseDisabled,
+    preferences: Utils.useRequisitionPreferences,
```

Comment:
yeah. it's a perverse sticking to convention.
We've used that pattern throughout rather than taking the hooks directly. The original idea was to make it easier to use as a consumer - you just need the single hook. I'm wondering if it is now too clumsy and we should just refactor that..